tests/foreman/api/test_docker.py

```diff
@@ -1132,7 +1132,7 @@ def setUpClass(cls):
         cls.cr_external = entities.DockerComputeResource(
             name=gen_string('alpha'),
             organization=[cls.org],
-            url=settings.docker.external_url,
+            url='http://{0}:2375'.format(settings.server.hostname),
```

Comment:
By doing that you are expecting that all satellites will have a local docker. The configuration was placed to skip this assumption and let the user decide what to do. Also you're assuming a port which also can change from setup to setup.

What we can do though is to skip the test if the external information is not available.